### PR TITLE
Updated the changelog

### PR DESCRIPTION
**Pull request by Rackover**
The changelog didn't show neither 3694 nor 3695. Even if they didn't alter much in the end, they shoud _at least_ be mentioned.
Feel free to correct my text if I made errors or typos, I'm not a native english speaker and I'm not sure of who contributed or not to those patches.

### CODE REVIEW

File: changelog.md [hunk 1]
```diff
@@ -1,3 +1,22 @@
...
+- CookieNoob
+
+Patch 3694 (21st December, 2017)
+============================
+### Other
+- Added christmas-related content, like reclaimable gifts.
+- Renamed the Galactic Colossus accordingly
+
+### Contributors
+- IceDreamer
```
Comment (CookieNoob) [inline changelog.md]: considering that ozonex and I for the most part did the christmas changes, I'm not sure why you mention icedreamer here and not us...

Comment (Rackover) [reply changelog.md]: I just checked the commits and they were attributed to aeoncleanse and cookieNoob. Now if you feel like this changelog contributor's list isn't fair, as I said, feel free to correct or edit it.

Comment (CookieNoob) [reply changelog.md]: it should be OzoneX and CookieNoob for the christmas patch ;)

**Comment by aeoncleanse** [comment]
Merged d8805888982c3395f3c102214148f5b395b6aaff

**Pull request closed without merging**